src/common/constants.ts

```diff
@@ -8,4 +8,5 @@ export const METAMASK_SNAPS_URL = 'https://snaps.metamask.io/'
 export const METAMASK_SNAPS_EXTERNAL_URL = 'https://metamask.io/snaps/'
 export const METAMASK_PORTFOLIO_URL = 'https://portfolio.metamask.io/'
 
-export const DEV_PORTAL_URL = 'https://developers.hyperplay.xyz/'
+export const DEV_PORTAL_URL =
```

Comment:
adding note to use production when https://github.com/valist-io/hyperplay-dev/pull/180 is merged